### DISCUSSION TITLE
我就是说，动态磁贴有没有实行的可能😊

### DISCUSSION THREAD

**Discussion by KIITTID** [ywmoyue/biliuwp-lite#209] [feature]
RT

**Comment by ywmoyue** [comment]
win11都没磁贴了，能调出磁贴菜单的explorer patcher看起来也不维护了，还加磁贴干嘛

**Reply by KIITTID** [reply]
照顾一下WIN10用户咯，还有没有我用[ExplorerPatcher](https://github.com/valinet/ExplorerPatcher)还原了WIN10开始菜单😊

**Reply by KIITTID** [reply]
大佬有没有可能加呀，有可能加的话我就继续把4*2的磁贴开着🙂

**Reply by ywmoyue** [reply]
暂时不会搞
